discord/connection.go

```diff
@@ -31,7 +32,7 @@ const (
 	ConnectionTypeSteam              ConnectionType = "steam"
 	ConnectionTypeTikTok             ConnectionType = "tiktok"
 	ConnectionTypeTwitch             ConnectionType = "twitch"
-	ConnectionTypeTwitter            ConnectionType = "twitter"
+	ConnectionTypeX                  ConnectionType = "twitter"
```

Comment:
@topi314 do we want to break this? I don't think many people use it anyway, but I'm not sure

Reply:
uh, we could add an alias to have both, I don't really care tbh

Reply:
![](https://cdn.discordapp.com/emojis/1009421361657155705.png?size=64)